app.js

```diff
@@ -1,25 +1,31 @@
-const express = require ("express")
-const app = express()
-const { getTopics } = require("./controllers/topics.controllers")
-const endpoints = require ("./endpoints.json")
-const { getArticleById } = require("./controllers/articles.controllers")
- 
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics.controllers");
+const endpoints = require("./endpoints.json");
+const { getArticleById, getArticles } = require("./controllers/articles.controllers");
 
-app.use(express.json())
-
-
-app.get("/api/topics", getTopics)
+app.use(express.json());
 
 app.get("/api", (request, response) => {
-    response.status(200).send({endpoints: endpoints})
-})
+    response.status(200).send({ endpoints: endpoints });
+});
 
-app.get("/api/articles/:article_id", getArticleById)
+app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticleById);
 
-app.all("*", (request, response, next) => {
-    response.status(404).send({message: "URL NOT FOUND"})
+app.get("/api/articles", getArticles);
+
+app.use((err, request, response, next) => {
```

Comment:
This is probably 2x instances of app.use method, separate the custom errors and invoke next if there is no err.status property